pkg/Microsoft.Windows.Compatibility/tests/publishcompatibilityassets.ps1

```diff
@@ -47,34 +39,25 @@ if (!(Test-Path $localPackageSourcePath))
 	}
 }
 
-$restoreSources = -join("https://dotnet.myget.org/F/aspnetcore-dev/api/v3/index.json;https://api.nuget.org/v3/index.json;https://dotnet.myget.org/F/dotnet-core/api/v3/index.json;https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json;", $localPackageSourcePath)
+$restoreSources = -join("https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json;https://dotnetfeed.blob.core.windows.net/aspnet-aspnetcore/index.json;", $localPackageSourcePath)
 
 $compatPackageVersion = _getPackageVersion "Microsoft.Windows.Compatibility"
-$privatePackageVersion = _getPackageVersion "Microsoft.Private.CoreFx.NETCoreApp"
-
-Write-Output "Calling dotnet restore"
-& $dotnetPath restore --packages $packagesCachePath /p:RestoreSources="$restoreSources" /p:TargetFramework=$targetFramework /p:CompatibilityPackageVersion=$compatPackageVersion /p:PrivateCorefxPackageVersion=$privatePackageVersion /p:RuntimeIdentifiers=$rid $csprojPath
```

Comment:
It’s a bummer this plumbing needs to be here.  Wish we could just do it once and have it work for any package we build.

Reply:
That said it's existing and not something you need to tackle at the moment, but consider it the next time you touch the set of packages that this references.

Reply:
Instead of doing all this stuff in this script, wouldn't make sense to have a target in the csproj that gets the version of the packages from the package reports? We already download the package reports from the artifacts, then we could just call the publish target and hook up our target to get the versions somehow?

Reply:
To do in a target it would need to be an inital-target that runs before restore.